docs/Adding-Software-to-Homebrew.md

```diff
@@ -313,47 +311,45 @@ Examples of difficult, unclear commit summaries:
 
 #### Pushing
 
-Push your changes from the branch `my-new-cask-branch` to your GitHub account:
+Push your changes on the branch `my-new-cask-branch` to your GitHub account:
 
 ```bash
-$ git push {{my-github-username}} my-new-cask-branch
+git push {{my-github-username}} my-new-cask-branch
 ```
 
-If you are using [GitHub two-factor authentication](https://help.github.com/articles/about-two-factor-authentication/) and set your remote repository as HTTPS you will need to set up a personal access token and use that instead of your password. Further information [here](https://help.github.com/articles/https-cloning-errors/#provide-access-token-if-2fa-enabled).
+If you are using [GitHub two-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa) and have set your remote repository as HTTPS you will need to [set up a personal access token]((https://docs.github.com/en/repositories/creating-and-managing-repositories/troubleshooting-cloning-errors#provide-an-access-token)) and use that instead of your password.
```

Comment:
```suggestion
If you are using [GitHub two-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa) and have set your remote repository as HTTPS you will need to [set up a personal access token](https://docs.github.com/en/repositories/creating-and-managing-repositories/troubleshooting-cloning-errors#provide-an-access-token) and use that instead of your password.
```